bindings/protocols/coap/index.html

```diff
@@ -339,20 +361,40 @@ <h3>Observing Resources</h3>
+                Consequently, a consumer can always fall back to polling a resource if observing that resource is permanently or temporarily not
                 possible.
                 In fact, a typical CoAP client implementation is to poll for as long as the client is interested in having a current
-                representation, setting the Observe option each time.
+                representation, setting the `Observe` option each time.
                 As long as the client is receiving notifications, it doesn't need to poll; as soon as the notifications are missing,
-                the client polls again with the Observe option set.
+                the client polls again with the `Observe` option set.
+            </p>
+            <p>
+                The `unobserveproperty` and `unsubscribeevent` operations with CoAP observe MUST be performed as specified in section 3.6 of [[RFC7641]].
+                Note that this operation can be performed either actively (i.e., by sending an explicit deregistration request) or passively (by "forgetting" the observe relation and sending a reset message after receiving the next notification response from the server).
+                For the `unobserveproperty` and `unsubscribeevent` operations to work, they MUST refer to the same resource as the matching `observeproperty` and `subscribeevent` operations.
+                It is therefore RECOMMENDED to always include these operations in the same form as their counterparts when defining them for an affordance.
+                If no `unobserveproperty` or `unsubscribeevent` operation is defined for an observable property or an event, it SHOULD be assumed that the "passive" deregistration variant is meant to be used with the respective affordance.
+            </p>
+            <p>
+                The same considerations apply to the operations `observemultipleproperties`, `observeallproperties`, `subscribemultipleevents`, `subscribeallevents`, and their respective `unobserve-` and `unsubscribe-` counterparts.
+                Note, however, that it is NOT RECOMMENDED to use the `observeallproperties` and `subscribeallevents` operations with constrained devices that provide many properties and/or events, as the observe notifications may become too large for constrained devices or their surrounding environments to handle.
+            </p>
+            <p class="note" title="Advanced Event Mechanisms">
+                In the case of Event Affordances, CoAP Observe only supports a simply event notification mechanism that corresponds with the `data` field of the `EventAffordance` class within a Thing Description.
+                More complex event mechanisms may be supported via the procedures defined in specifications such as [[?I.D.-coap-pubsub]] or [[?I.D.-stp]] in the future.
```

Comment:
Section 3 should be a non-normative overview of the protocol; it should not contain any normative requirements. I would therefore propose to move this text to section 6.1, since it's actually providing more details on the default binding and not on the protocol. Apart from that, this PR looks good to me and can be merged.

Reply:
Thank you for your feedback! I moved the text containing assertions in 20847985c86304e82903d718e7a0c84dc73f7cc7.